### PR TITLE
Add config for Lenovo ThinkPad X1 Carbon

### PR DESCRIPTION
**Pull request by gaul**

### CODE REVIEW

File: configs/Lenovo/ThinkPad-X1-Carbon.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    ignore temp6
+    ignore temp7
+    ignore temp8
+    ignore temp9
+    ignore temp10
+    ignore temp12
+    ignore temp13
+    ignore temp14
+    ignore temp15
+    ignore temp16
```
Comment (olysonek) [inline configs/Lenovo/ThinkPad-X1-Carbon.conf:17]: Please add a comment explaining why you're ignoring all these temperatures. Are they always zero? Do they report nonsensical values? That's the kind of info that I'm looking for.

Other than that, it looks good to me. Thanks!

Comment (gaul) [reply configs/Lenovo/ThinkPad-X1-Carbon.conf:17]: Documented as always zero.

**Comment by olysonek** [comment]
Thank you!

**Comment by jnweiger** [comment]
Exactly the same config also applies for Lenovo Thinkpad 440s and 480s, should we create duplicates of thee X1 Carbon file?

**Comment by olysonek** [comment]
@jnweiger I guess. I don't see a better option.

**Pull request merged**